.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "next/core-web-vitals",
```

Comment:
this works with all the rest? prettier? rtl? etc.. ?

Reply:
yup added them back (had to disable no-default-import for /pages else nextJS complains)